### PR TITLE
 Update ready for release v2.0.0

### PR DESCRIPTION
**Pull request by matt-FFFFFF**
## This PR fixes/adds/changes/removes

1. Updates to documentation and module references for v2.0.0

## As part of this Pull Request I have

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues), for tracking and closure.
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located in the [Enterprise-Scale repo](https://github.com/Azure/Enterprise-Scale) in the directory: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/[Variables]-configure_connectivity_resources.md [hunk 1]
```diff
@@ -535,7 +662,117 @@ List of [Azure] resource IDs used to identify spoke Virtual Networks associated
 
 ### Configure hub networks (Virtual WAN)
 
-_Not implemented yet, coming soon._
+Define zero or more VWAN hub networks as a list of objects, each containing configuration values covering an address space, DDOS protection plan, expressroute gateway, vpn gateway and Azure firewall.
```
Comment (krowlandson) [inline docs/wiki/[Variables]-configure_connectivity_resources.md:665]: DDoS Protection plan is setup outside of the VWAN, and we actually don't have a means by which to link it to Virtual Hubs currently.

Comment (matt-FFFFFF) [reply docs/wiki/[Variables]-configure_connectivity_resources.md:665]: Cool - will change

**Review by krowlandson — APPROVED** [review]
🚀

**Pull request merged**